.github/workflows/ui_test.yml

```diff
@@ -0,0 +1,58 @@
+name: testing workflow
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: ikalnytskyi/action-setup-postgres@v5
+      - name: Postgres setup
```

Comment:
```suggestion
      - name: Postgres setup
        uses: ikalnytskyi/action-setup-postgres@v5
```

Reply:
Since steps must include either uses or run, the above suggestion gives error because the step will contain both uses and run if the suggested edit is implemented.